docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+external modules.
+
+Changes in reusable workflows should be tested through application to
+ITK external modules in a user fork. The ideal repositories to use
+for testing will depend on the workflow changes under test.
+
+The [ITKSplitComponents repository](https://github.com/InsightSoftwareConsortium/ITKSplitComponents)
+is a good target for testing relatively
+simple changes. ITKSplitComponents is a small header-only external module
+with a minimum testing suite, Python wrappings, and an example Jupyter Notebook.
+Do the following to test against ITKSplitComponents or another ITK external module:
+1. Commit your changes to `ITKRemoteModuleBuildTestPackageAction` and
+   push to make them available on your GitHub user account:
+```sh
+> git commit
+> git push
+```
+
+2. Fork the [ITKSplitComponents repository](https://github.com/InsightSoftwareConsortium/ITKSplitComponents)
+    to your GitHub user account and clone to your local machine:
+```sh
+> cd ..
+> git clone git@github.com:<your-username>/ITKSplitComponents.git
+> cd ITKSplitComponents
+```
+
+3. Create a development branch for your proposed changes:
+```sh
+> git checkout -b "my-cool-feature"
+```
+
+4. Update the external module workflow `.yml` file to reference your changes:
+```yml
+name: Build, test, package
+
+on: [push,pull_request]
+
+jobs:
+  cxx-build-workflow:
+    uses: <your-username>/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@<your-dev-branch>
+    with:
+      itk-cmake-options: '-DITK_BUILD_DEFAULT_MODULES:BOOL=OFF -DITKGroup_Core:BOOL=ON'
+
+  python-build-workflow:
+    uses: <your-username>/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-package-python.yml@<your-dev-branch>
+    secrets:
+      pypi_password: ${{ secrets.pypi_password }}
+```
+
+5. Commit and push your changes:
+```sh
+> git commit
+> git push
+```
+
+6. Visit `https://github.com/<your-username>/ITKSplitComponents/actions` to view running GHA jobs.
+
+Other ITK external modules may also be used for testing:
+- [ITKBSplineGradient](https://github.com/insightSoftwareConsortium/ITKbsplinegradient) can be
+  used for building with a dependency on one other ITK external module;
+- [ITKUltrasound](https://github.com/KitwareMedical/ITKUltrasound) can be used for building with
+  dependencies on multiple ITK external modules.
+
+## Creating a Pull Request
+
+You can propose changes to the central `InsightSoftwareConsortium` repository by
+creating a
+[pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+
+1. Visit `https://github.com/<your-username>/ITKRemoteModuleBuildTestPackageAction/pulls` and
+   click the green "New Pull Request" button to create a pull request for merging changes
+   from your development branch into `InsightSoftwareConsortium/main`.
+2. Describe your changes and create the pull request.
```

Comment:
Maybe link to https://github.com/InsightSoftwareConsortium/ITK/blob/23da3dfa79152648f3d38050ee147df47f41ce0a/CONTRIBUTING.md#commit-messages as guidance on what the ITK ecosystem commit messages should contain.